gitconfig

```diff
@@ -1,6 +1,6 @@
 [user]
-	name = David McCann
-	email = david.a.mccann@gmail.com 
+	name = Jeff Wishnie
```

Comment:
Any thoughts as to how to cleanly make this value parameterizable? Not exactly fork-friendly if you have to remember to change this back whenever you make a PR.